docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,140 @@
+Once your endpoint is created, you can test sending events to it from within the `Testing` tab of the  Endpoints Dashboard. 
+
+<Figure>
+  <img
+    src="/images/docs/webhooks/webhooks-endpoint-create.png"
+    alt="Share  webhooks endpoint Dashboard"
+  />
+</Figure>
+
+In order to disable an endpoint from the dashboard, you will select it and then click the kebab menu at the top right of the frame.  This option gives you the ability to disable the endpoint so that you may maintain the data provided, while no longer recieving messages. 
+
+<Figure>
+  <img
+    src="/images/docs/webhooks/webhooks-replay.png"
+    alt="Share replay and disable"
+  />
+</Figure>
+
+Before sending events, you should disable CSRF protection for the endpoint if it is enabled by default, and use the steps outlined in the ##Verifying Section to ensure the security of the message.  
+
+If the need arises, you can rotate the signing secret from within the Endpoint portion of the dashboard by clicking on the dropdown menu located above the item named Signing Secret.  Rotating the secret will sign the message with the new secret and old secret for a period of 24 hours. This method of roation gives you time to update the secret on the client side, without losing any event driven data.
+
+<Figure>
+  <img
+    src="/images/docs/webhooks/webhooks-advanced-configuration.png"
+    alt="Share advanced configuration"
+  />
+</Figure>
+
+In the event that your service needs custom headers to be set in order to work with load balancers or application servers, the Advanced section of the Endpoint configuration provides the option to configure these values, alongside custom rate limiting. 
+
+From within the dashboard you also have the ability to filter events by date and eventType.  This feature is nested within the Logs tab of the dashboard and can be seen below. 
+
+<Figure>
+  <img
+    src="/images/docs/webhooks/webhooks-filter.png"
+    alt="Share filtering capability"
+  />
+</Figure>
+
+### Verifying Webhooks
+Webhooks can have the ability to generate security holes, but with svix verification features this can be mitigated.  When you verify your webhooks using svix, your protect agains man-in-the-middle, CSRF, and replay attacks. Because of this, the importance of verification cannot be understated, and should be considered paramount in your integration.  For a deeper dive into why you should verify webhooks, the [Svix Documentation](https://docs.svix.com/receiving/verifying-payloads/why) is a great resource.
+
+In order to validate a webhook with the svix libraries run: 
+```
+npm install svix
+```
+Then, inside of your application where you call the webhook:
+```typescript
+import { Webhook } from "svix";
+
+const secret = "<your signing secret>";
```

Comment:
Should we display it like we do in code snippets where we mention the secret key?

Reply:
Do you mean like below: 
```
import { createClient } from "@liveblocks/client";

const client = createClient({
  publicApiKey: "pk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
});
```
and then this would then become 
`const secret = "xxxxxxxxx"`
for however many characters the client secret is? AFAIK it is not prefixed like our publicAPIKey or similar.

Reply:
Yes... but not sure if it applies. Maybe something worth asking @marcbouchenoire since he designed those components.

FYI Marc, with Svix we're introducing a new concept of secret key that doesn't come from Liveblocks. What do you think we should do with it?

Reply:
I'm not sure about having tooltips for non-Liveblocks keys since authoring them could be difficult (across two repos, and I'm not sure we want to keep a list of these specific cases outside of their uses) compared to the value they would bring (no extra "magic" info or links), I think using only the same redacting style with `x`s would be enough to fit in.

Reply:
thanks @marcbouchenoire

Reply:
The key will "come from liveblocks" in the sense that it will be accessible from the integrated dashboard in liveblocks.io